consensus/polybft/README.md

```diff
@@ -5,153 +5,109 @@ Polybft is a consensus protocol, which runs [go-ibft](https://github.com/0xPolyg
-    $ polygon-edge polybft register-validator --data-dir ./test-chain-1 \
-    --supernet-manager <address_of_SupernetManager_contract>
-    ```
-
-10. Initial staking on rootchain - each validator needs to do initial staking on rootchain (StakeManager) contract. **This command is for testing purposes only.**
-
-    ```bash
-    $ polygon-edge polybft stake --data-dir ./test-chain-1 --supernet-id <supernet_id_from_genesis> \
-    --amount <amount_of_tokens_to_stake> \
-    --stake-manager <address_of_StakeManager_contract> --stake-token <address_of_erc20_token_used_for_staking>
-    ```
-
-11. Do mint and premine for relayer node. **These commands should only be executed if non-mintable erc20 token is used**
-
-    ```bash
-    $ polygon-edge bridge mint-erc20 \ 
+    $ blade mint-erc20 \ 
     --erc20-token <address_of_native_root_erc20_token> \
     --private-key <hex_encoded_private_key_of_token_deployer> \
     --addresses <address_of_relayer_node> \
     --amounts <ammount_of_tokens_to_mint_to_relayer>
     ```
 
      ```bash
-    $ polygon-edge rootchain premine \ 
+    $ blade rootchain premine \ 
        --erc20-token <address_of_native_root_erc20_token> \
-       --supernet-manager <address_of_CustomSupernetManager_contract_on_root> \
+       --blade-manager <address_of_BladeManager_contract_on_root> \
        --private-key <hex_encoded_private_key_of_relayer_node> \
-       --amount <ammount_of_tokens_to_premine>
+       --premine-amount <ammount_of_tokens_to_premine>
+       --stake-amount <ammount_of_tokens_relayer_staked_on_blade>
     ```
 
-12. Finalize genesis validator set on rootchain (SupernetManager) contract. This is done after all validators from genesis do initial staking on rootchain, and it's a final step that is required before starting the child chain. This needs to be done by the deployer of SupernetManager contract (the user that run the deploy command). He can use either its hex encoded private key, or data-dir flag if he has secerets initialized. If enable-staking flag is provided, validators will be able to continue staking on rootchain. If not, genesis validators will not be able update its stake or unstake, nor will newly registered validators after genesis will be able to stake tokens on the rootchain. Enabling of staking can be done through this command, or later after the child chain starts.
+5. Finalize bridge setup on rootchain (BladeManager) contract. This is done after all addresses we need premined on rootchain, and it's a final step that is required before starting the child chain. This needs to be done by the deployer of BladeManager contract (the user that run the deploy command). He can use either its hex encoded private key, or data-dir flag if he has secerets initialized. The commands says to rootchain that tokens of premined addresses will be locked on L1, so that they can be used from L2 start. **This command should only be executed if non-mintable (L1 originated) erc20 native token is used**
```

Comment:
```suggestion
5. Finalize bridge setup on rootchain (`BladeManager`) contract. This is done after all addresses we need premined on rootchain, and it's a final step that is required before starting the child chain. This needs to be done by the deployer of BladeManager contract (the user that run the deploy command). He can use either its hex encoded private key, or data-dir flag if he has secerets initialized. The commands says to rootchain that tokens of premined addresses will be locked on L1, so that they can be used from L2 start. **This command should only be executed if non-mintable (L1 originated) erc20 native token is used**
```

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/215/commits/685b8dcee2ed1442d546538e48f1067abbee2e26